client/human/GGHumanClientApp.cpp

```diff
@@ -799,7 +799,7 @@ void GGHumanClientApp::LoadSinglePlayerGame(std::string filename/* = ""*/) {
     // leving setup_data.m_players empty : not specified when loading a game, as server will generate from save file
 
 
-    m_networking->SendMessage(HostSPGameMessage(setup_data));
+    m_networking->SendMessage(HostSPGameMessage(setup_data, DependencyVersions()));
```

Comment:
I guess this is only relevant if launching a stand alone server locally that was built against different dependencies than the client? Normally they should be consistent if built together.

Reply:
It possible to use `network.server.external.force` and connect somewhere else.